lucene/analysis/opennlp/src/test/org/apache/lucene/analysis/opennlp/TestOpenNLPChunkerFilterFactory.java

```diff
@@ -58,7 +58,7 @@ public class TestOpenNLPChunkerFilterFactory extends BaseTokenStreamTestCase {
     8, 15, 17, 21, 23, 29, 30, 39, 46, 48, 49, 51, 57, 58
   };
   private static final String[] SENTENCES_chunks = {
-    "B-NP", "I-NP", "I-NP", "B-VP", "B-NP", "I-NP", "O", "B-NP", "I-NP", "I-NP", "O", "B-NP",
+    "B-NP", "I-NP", "I-NP", "I-NP", "I-NP", "I-NP", "O", "B-NP", "I-NP", "I-NP", "O", "B-NP",
```

Comment:
This change so far is naively what makes the tests pass ... however

```suggestion
    "B-NP", "I-NP", "I-NP", "I-VP", "I-NP", "I-NP", "O", "B-NP", "I-NP", "I-NP", "O", "B-NP",
```

i.e. `"I-VP"` for `"has"` would seem more correct, no?

Reply:
> ... would seem more correct, no?

Upon further consideration, the _"Tagging models are created from tiny test data in opennlp/tools/test-model-data/ and are not very accurate."_ comment from line 31-32 above applies here probably.